virtualmachine/vsphere/vm.go

```diff
@@ -175,9 +175,9 @@ type ClusterComputeResource struct {
 	FreeMemory     int64        `json:"free_memory"`
 	TotalStorage   int64        `json:"total_storage"`
 	FreeStorage    int64        `json:"free_storage"`
-	NoOfHosts      int          `json:"number_of_hosts"`
-	NoOfDatastores int          `json:"number_of_datastores"`
-	NoOfNetworks   int          `json:"number_of_networks"`
+	NoOfHosts      int          `json:"number_hosts"`
```

Comment:
Are these fields not used so far? Do we require corresponding change at other places?